cpp/src/hash/hash_allocator.cuh

```diff
@@ -20,9 +20,9 @@
 #include <new>
 
 #include <rmm/rmm.h>
-#include <rmm/mr/device_memory_resource.hpp>
-#include <rmm/mr/managed_memory_resource.hpp>
-#include <rmm/mr/default_memory_resource.hpp>
+#include <rmm/mr/device/device_memory_resource.hpp>

```

Comment:
It looks like the CI build is failing here because it doesn't have the changes from https://github.com/rapidsai/rmm/pull/262 yet. Does this come from conda `librmm:            0.13.0a200124-cuda10.0_61 rapidsai-nightly`?

Reply:
So it _used_ to be that if you had two PRs with the same branch name between cudf and RMM that CI would automatically pull the PR from RMM.